sdk/dataprotection/Azure.ResourceManager.DataProtectionBackup/api/Azure.ResourceManager.DataProtectionBackup.netstandard2.0.cs

```diff
@@ -1237,6 +1285,7 @@ public partial class DataProtectionBackupVaultProperties
         public Azure.ResourceManager.DataProtectionBackup.Models.BackupVaultFeatureSettings FeatureSettings { get { throw null; } set { } }
         public bool? IsVaultProtectedByResourceGuard { get { throw null; } set { } }
         public Azure.ResourceManager.DataProtectionBackup.Models.DataProtectionBackupProvisioningState? ProvisioningState { get { throw null; } }
+        public System.Collections.Generic.IList<string> ReplicatedRegions { get { throw null; } }
```

Comment:
are the items here `azure locations`? if so we could change their format to `azure-location`

Reply:
added the correct formats